### PR TITLE
refactor: Rename `get_up_to_date_prefix` to `update_prefix`

### PR DESCRIPTION
**Pull request by Hofer-Julian**
The return parameter was never used

### CODE REVIEW

File: src/environment.rs [hunk 1]
```diff
@@ -247,19 +247,18 @@ impl LockFileUsage {
...
 /// takes up a lot of memory and takes a while to load. If `sparse_repo_data` is
 /// `None` it will be downloaded. If the lock-file is not updated, the
 /// `sparse_repo_data` is ignored.
-pub async fn get_up_to_date_prefix(
+pub async fn update_prefix(
     environment: &Environment<'_>,
     lock_file_usage: LockFileUsage,
     mut no_install: bool,
-) -> miette::Result<Prefix> {
+) -> miette::Result<()> {
```
Comment (baszalmstra) [inline src/environment.rs:261]: I can imagine that it would still be useful to have the prefix. I don't see a reason to remove it.

**Review by baszalmstra — APPROVED** [review]
One comment but LGTM.

File: src/environment.rs (hunk 1 above)
Comment (Hofer-Julian) [reply src/environment.rs:261]: But that is the whole point of the PR 😅

I am fine with closing it though.

Comment (baszalmstra) [reply src/environment.rs:261]: I think the name change is still valid though. With a get_ function I would expect a user to also use the return value.

Comment (baszalmstra) [reply src/environment.rs:261]: I think `update_lockfile` actually makes more sense then 😄

Comment (Hofer-Julian) [reply src/environment.rs:261]: Fine by me :D

**Review by baszalmstra — APPROVED** [review]

**Pull request merged**